core/base/integralLines/IntegralLines.h

```diff
@@ -403,9 +403,9 @@ void ttk::IntegralLines::receiveElement(
 
   // Create integral line object on this process
   int threadNum{0};
-#ifdef TTK_ENABLE_OPENMP
+#ifdef TTK_ENABLE_OPENMP4
   threadNum = omp_get_thread_num();
```

Comment:
isn't this call supported by openmp3??

Reply:
Yes, that call is supported. But the since some operations later done the call chain, like on line 531 are not supported, I wanted to disable OpenMP for the entire call to be defensive. One is still using OpenMP at the top level, but not at the bottom of a call stack that might lead to errors right? 
But I have not studied the code very carefully here. Maybe openmp could still be enabled for some parts?

Reply:
good catch :)